StatusAPI.md

```diff
@@ -0,0 +1,245 @@
+# Candidate Status Documentation
+
+Contents
+
+- [Candidate Status Fields](#candidate-status-fields)
+- [Retrieve all statuses for a single email](#retrieve-all-statuses-for-a-single-email)
+- [Insert a status](#insert-a-status)
+- [Retrieve multiple emails or multiple requisition_ids](#retrieve-multiple-emails-or-multiple-requisition_ids)
+
+## Candidate Status Fields
+
+| Field | Req’d | Type | Comment |
+| ------ | ------ | ------ | ------ |
+| account | Y | String(20) | CB Account DID |
+| candidate_email | Y | String(255) | 255 is not a typo. |
+| vendor_key  | N | String(20) | Type of source system(Bullhorn, Luceo, TEE…) |
+| client | N | String(20) | System calling this service (MyCandidates, CandidateStream) |
+| requisition_id | N | String(1024) | Job this status applies to (can be null) |
+| requisition_title | N | String(256) |  |
+| level_id | N | String(64) | ID of Candidate/Application |
+| status | Y | String(100) | The raw status as recorded by source system |
+| status_date | Y | Date/time | UTC Date/time of status changeFormat: YYYY-MM-DD THH:MM:SSZ 1776-07-04T19:04:01Z |
+| status_level | Y | String(20) | Candidate/Application |
+| status_detail | N | String(4096) | Free form string  |
+
+#### Unique Key:
+  - account
+  - candidate_email
+  - vendor_key
+  - requisition_id
+  - status_level
+  
+This assumes that we are:
```

Comment:
This section is not clear. It sounds like you're trying to explain which statuses can go with which vendors. I would use a notation to make this clearer - instead of "and" I would use |. Also if you use a category of action types, identify it as an action type and give a clear list of what each category means in each description